docs/src/Nlay_tutorial.md

```diff
@@ -0,0 +1,151 @@
+# N-layer Cylinder Examples
+
+A major functionality of this package is the ability to perform forward modeling of the diffusion equation in arbitrarily layered media. This page will detail how to use the Nlayer code including the packages `fluence_DA_Nlay_cylinder_CW`, `fluence_DA_Nlay_cylinder_TD`, and `fluence_DA_Nlay_cylinder_FD`. There are also other ppackages for Nlayerd cylinders such as `flux_DA_Nlay_cylinder_CW` that can be used in a similar manner.
```

Comment:
Let's refer to `fluence_DA_Nlay_cylinder_CW` and others as functions instead of packages. Also probably good to hyphenate N-layer

Reply:
Ppackages is mispelled on last line

Reply:
Fixed